test/testcafe/spec/EnrollAuthenticatorDataPhoneView_spec.js

```diff
@@ -116,15 +120,15 @@ test.requestHooks(voiceOnlyOptionMock)('default is voice mode', async t => {
   await t.expect(enrollPhonePage.getSaveButtonLabel()).eql('Receive a code via voice call');
 
   // Extension field is not hidden
-  await t.expect(enrollPhonePage.extensionIsHidden()).eql(false);
+  await t.expect(await enrollPhonePage.extensionIsHidden()).eql(false);
   // Phone field is rendered small
   await t.expect(enrollPhonePage.phoneNumberFieldIsSmall()).eql(true);
```

Comment:
nit: 
```suggestion
  // Add comment here saying that this is only applicable to v2
  // since v3 uses flexbox to render phone and extension fields on same line.
  if (!userVariables.v3) {
    await t.expect(enrollPhonePage.phoneNumberFieldIsSmall()).eql(true);
  }
```